airbyte-integrations/connectors/source-looker/source_looker/client.py

```diff
@@ -27,15 +27,16 @@
 
 import backoff
 import requests
+from airbyte_protocol import AirbyteStream
 from base_python import BaseClient
 from requests.exceptions import ConnectionError
 from requests.structures import CaseInsensitiveDict
-
+from typing import Generator
 
 class Client(BaseClient):
     API_VERSION = "3.1"
 
-    def __init__(self, domain: str, client_id: str, client_secret: str):
+    def __init__(self, domain: str, client_id: str, client_secret: str, run_look_ids: list=[]):
```

Comment:
for future readers, can we have a docstring summarising why this constructor needs to dynamically generate schemas here and link to the relevant functions?